petclinic-frontend/src/features/customers/components/AddPetForm.css

```diff
@@ -77,7 +77,7 @@
 }
 
 .cancel-form-button {
-  background-color: #007BFF !important;
+  background-color: #007bff !important;
```

Comment:
Usually the use of the !important keyword is frowned upon because it makes the code less maintanable. I suggest you create css modules for your css and find the root of why without the !important keyword your style isn't being applied.